dev/integration_tests/flutter_gallery/lib/demo/material/tabs_demo.dart

```diff
@@ -14,7 +14,7 @@ const String _kGalleryAssetsPackage = 'flutter_gallery_assets';
 class _Page {
   _Page({ this.label });
   final String label;
-  String get id => label[0];
+  String get id => label.characters.first;
```

Comment:
I believe this example does not exist in the [gallery repo](https://github.com/flutter/gallery), so there is nothing to update over there.